BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -220,17 +221,27 @@ def BuildFspBins (fsp_dir, sbl_dir, fsp_inf, silicon_pkg_name, flag):
 
     CopyFileList (copy_list, fsp_dir, sbl_dir)
 
+def DoAddPaddingData(File, SlotSize):
+  with open(File, "rb") as fd:
+    Binary = fd.read()
+    if len(Binary) > SlotSize:
+      print (("Microcode patch %s is larger than slot size(%d)\n") % (File, SlotSize))
+      assert (False)
+    elif len(Binary) < SlotSize:
+      Binary = Binary + b'\xff' * (SlotSize - len(Binary))
+
+  with open(File, "wb") as fd:
+    fd.write(Binary)
+
 def Main():
 
-    if len(sys.argv) < 3:
+    if len(sys.argv) < 4:
```

Comment:
What is the argument here?

Reply:
Here are the arguments given to this file, in order:
1) python file name for building binaries (e.g. C:\SblPlatform\SblOpen\BootloaderCorePkg\Tools\PrepareBuildComponentBin.py)
2) working directory (e.g. C:\SblPlatform\SblOpen)
3) silicon package name (e.g. TigerlakePkg)
4) build mode (e.g. /r) 
5) ucode slot size (e.g. 114688)